src/components/new-request/search.vue

```diff
@@ -566,16 +611,61 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
     this.business = business
     this.entity_type_cd = this.business?.legalType || null
     this.setCorpNum(business?.identifier || null)
+
+    if (this.isRestoration) {
+      // Check if not XPRO and BC restorable
+      if (!this.isSelectedXproAndRestorable && this.isBcRestorable) {
```

Comment:
- First conditional: If the selected business is restorable and BC based (BENs for example).
- Second Conditional: If the selected business is restorable but XPRO (Limited companies, LLCs for example).
- Third conditional (else statement): If the selected business is not restorable.

Reply:
Thanks for the code comments, too!